client/src/App.js

```diff
@@ -10,6 +10,14 @@ import Recipes from "./pages/recipe";
 import SearchFood from "./pages/searchfood";
 import Login from "./pages/login";
 import Contact from "./pages/contact";
+import Auth0Lock from 'auht0-lock';
+
+
+
+const lock = new Auth0Lock(
+    '7kHcCdykKkIirg50ElD91DIL2ON5O2Jb',
```

Comment:
I should probably replace these with env vars. So don't copy these. They *are not* secret so it's not bad that I checked them in, but they should probably be env vars of the form `REACT_APP_CLIENT_ID` and `REACT_APP_AUTH0_DOMAIN`. There are a few ways you can load config in the FE so Cristina's example may have a different way of doing this that does not use `REACT_APP` vars--I'm sure whatever she did will work so don't worry if she recommended something different.